cmdstanpy/cmdstan_args.py

```diff
@@ -277,6 +277,38 @@ def compose(self, idx: int, cmd: str) -> str:
         return cmd
 
 
+class GenerateQuantitiesArgs(object):
```

Comment:
`GenerateQuantitiesArgs` or `GeneratedQuantitiesArgs`

Reply:
We've followed naming as suggested by @mitzimorris in #67